### PR TITLE
[terra-aggregate-translations] Migrate terra-aggregate-translations

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

Move terra-aggregate-translations to terra-toolkit and scope under `@cerner`

### Additional Details

@cerner/terra

[CONTRIBUTORS.md]: ../blob/main/CONTRIBUTORS.md
[License]: ../blob/main/LICENSE

### CODE REVIEW

File: packages/terra-aggregate-translations/README.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+| configPath | -c, --config | String | The path to the terra i18n configuration file | undefined |
+| format | -f, --format | String | The format of syntax to output the translations with. Possible values are 'es5' and 'es6' | 'es5' |
+
+#### Setup Example
+
+The `aggregate-translations` setup function can be used as follows:
+
+```js
+// webpack config file
+const aggregateTranslations = require('terra-aggregate-translations');
```
Comment (jmsv6d) [inline packages/terra-aggregate-translations/README.md]:
```suggestion
const aggregateTranslations = require('@cerner/terra-aggregate-translations');
```

**Review by jmsv6d — APPROVED** [review]

File: packages/terra-aggregate-translations/README.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+```
+
+#### CLI Example
+
+The `aggregate-translations` CLI is supplied as a bin script, called `tt-aggregate-translations`, and can be used as follows:
+
+```js
+scripts: {
+    // ...other scripts
+    "aggregate-translations": "tt-aggregate-translations -l ['en','es'] -e ./node_modules/packageToExclude",
```
Comment (emilyrohrbough) [inline packages/terra-aggregate-translations/README.md:85]: Is there work planned to hook into terra cli?

File: packages/terra-aggregate-translations/README.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+The aggregate-translations script has the ability to compile with ES6 syntax by setting the format prop to 'es6'. To get these to work properly with Jest and the rest of your configuration, you need to add the [babel-plugin-syntax-dynamic-import](https://www.npmjs.com/package/babel-plugin-syntax-dynamic-import) plugin to your babel configuration. Since we are not using Babel 7 in Terra, you need to use version 6.18.0 to be compatible with our code base.
+
+## Usage with Rails/Webpacker
+
+If you are using Rails/Webpacker, you can generate the aggregate-translation files and ensure they resolve correctly when running webpack with the following config setup in `app/config/webpack/environment.js` file.
+
+```js
+const { environment } = require('@rails/webpacker')
+
+const aggregateTranslations = require('terra-aggregate-translations');
```
Comment (emilyrohrbough) [inline packages/terra-aggregate-translations/README.md]:
```suggestion
const aggregateTranslations = require('@cerner/terra-aggregate-translations');
```

File: packages/terra-aggregate-translations/UpgradeGuide.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Terra Aggregate Translations
+
+## Changes from version 3 to version 4
+
+### Project Name Updated with @cerner Scope
```
Comment (emilyrohrbough) [inline packages/terra-aggregate-translations/UpgradeGuide.md:5]: Mention dropping node 8 support?

**Review by emilyrohrbough — APPROVED** [review]

File: packages/terra-aggregate-translations/README.md (hunk 2 above)
Comment (ryanthemanuel) [reply packages/terra-aggregate-translations/README.md:85]: Logged this: https://github.com/cerner/terra-toolkit/issues/550

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**